src/Microsoft.PowerShell.Commands.Management/commands/management/WMIHelper.cs

```diff
@@ -162,7 +162,7 @@ internal override void StartOperation()
             }
 
             thread.IsBackground = true;
-            // thread.SetApartmentState( ApartmentState.STA);
+            thread.SetApartmentState(ApartmentState.STA);
```

Comment:
#if UNIX

Reply:
This source file is currently not even included in compilation.  I believe it's used by WMI cmdlets.

Reply:
I'd add the fix because later it will very difficult to find the place if need be.

Reply:
I'll add it since it's a small change but there's not plan to bring back the WMI cmdlets, but who knows how things change in the future :)